CHANGES.rst

```diff
@@ -1,6 +1,18 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-19)
+--------------------------
+
+- Moves workflow engine to the same Kubernetes pod with the REANA-Job-Controller
+  (sidecar pattern).
+- Allows to set compute backend, name and Kerberos authentication requirement
```

Comment:
The same comment for Condor/Slurm and item list order as in https://github.com/reanahub/reana-workflow-engine-cwl/pull/129/files#r359968165